### PR TITLE
Add empty state

### PR DESCRIPTION
**Pull request by PanSpagetka**
https://issues.redhat.com/browse/RHCLOUD-12682
Screenshot from 2021-04-14 11-08-08

### CODE REVIEW

File: src/store/default-state.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
+export const defaultRecomendations = [
```
Comment (Hyperkid123) [inline src/store/default-state.js:1]: I will make you sad, but these are (maybe with different content) already in `default-content-config.json`

File: src/store/default-state.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        action: {
+          title: 'Learn More',
+          href:
+            'https://www.redhat.com/en/technologies/cloud-computing/openshift',
+        },
+      },
+    ],
+  },
+  {
+    title: 'Insights for RHEL',
```
Comment (epwinchell) [inline src/store/default-state.js:77]: @PanSpagetka This should be fully spelled out as of Friday afternoon: "Insights for Red Hat Enterprise Linux"

File: src/store/default-state.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        action: {
+          title: 'Get Started',
+          href:
+            'https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-api-management',
+        },
+      },
+    ],
+  },
+  {
+    title: 'Insights for Ansible',
```
Comment (epwinchell) [inline src/store/default-state.js:30]: @panspagetka Same here "Insights for Ansible Automation Platform"

**Review by epwinchell — APPROVED** [review]

File: src/utils/default-content-config.json [hunk 4]
```diff
@@ -4,85 +4,123 @@
...
-          "method": "isNotEntitled",
-          "args": ["rhel"]
-        }],
+      "items": [
+        {
+        "id": "remediations-default-1",
         "icon": "play",
         "state": "success",
-        "title": "Red Hat Insights for Red Hat Enterprise Linux simplifies how IT teams maintain and optimize a stable, secure, and performant operating environment.",
+        "title": "Try Streams for Apache Kafka.",
```
Comment (Hyperkid123) [inline src/utils/default-content-config.json:12]: I think this should be visible before 26th. Can we comment it out for now?

**Review by Hyperkid123 — CHANGES_REQUESTED** [review]

File: src/utils/default-content-config.json (hunk 4 above)
Comment (PanSpagetka) [reply src/utils/default-content-config.json]: I had to remove it, because it is JSON not javascript.

Comment (Hyperkid123) [reply src/utils/default-content-config.json]: Right ofc.

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**